psi4/driver/procrouting/proc_table.py

```diff
@@ -113,7 +113,6 @@
         'fci'           : proc.run_detci,
         'casscf'        : proc.run_detcas,
         'rasscf'        : proc.run_detcas,
-        'adc'           : proc.run_adc,
```

Comment:
To prevent a deprecation can you add something like:

```python
'adc'           : proc.adc_deprecation,

def adc_deprecation(*args, **kwargs):
     warnings.warn("The method 'adc' has been deprecated please use .... The method 'adc' will be deprecated in Psi4 1.6.", DeprecationWarning)
     return proc.select_adc2(*args, **kwargs)
```

@loriab Do we wrap deprecation warnings these days?

Reply:
Thanks I fixed that.